mp/game/bin/base.fgd

```diff
@@ -4800,6 +4883,7 @@
 
     // Inputs
     input Teleport(void) : "Teleport the target entity."
+    input TeleportEntity(string) : "Teleport the output parameter box entity."
```

Comment:
Wouldn't it be an input parameter?

Reply:
It's the parameter override of an output in hammer.